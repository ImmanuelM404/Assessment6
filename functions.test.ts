const {shuffleArray} = require('./utils')

describe('shuffleArray should', () => {
    // test skelton " test('should run', () => expect().toBe(//.toBe is the method, in () should be expected return value))
    test('should run', () => { 
        expect(shuffleArray).toHaveLength(1)
    })
    test('should run', () => { 
        expect.arrayContaining(shuffleArray)
    })
    test('should run', () => { 
        expect(shuffleArray).toMatchSnapshot()
    })
})