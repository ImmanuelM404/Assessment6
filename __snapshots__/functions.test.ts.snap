// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shuffleArray should should run 1`] = `[Function]`;
